File: sdkwork-backend-sdk-python/sdkwork_backend_sdk/models/plus_api_key_form.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class PlusApiKeyForm:
    """API key creation form"""
    name: str
    key_value: str
    type: str
    status: str = None
    expire_time: str = None
    description: str = None

File: sdkwork-backend-sdk-go/types/plus_api_key_form.go
package types

// API key creation form
type PlusApiKeyForm struct {
	Name string `json:"name"`
	KeyValue string `json:"keyValue"`
	Type string `json:"type"`
	Status string `json:"status"`
	ExpireTime string `json:"expireTime"`
	Description string `json:"description"`
	LastUsedTime string `json:"lastUsedTime"`
}
